### ISSUE TITLE
Accessibility: CCM Dashboard contrast colors

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#632]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Critical

Feature: CCM dashboard

Issue notes:
The grey text on the grey background used for the tiles does not provide sufficient contrast (3.9:1; should be at least 4.5:1) 

Criterion:
1.4.3 Contrast (Minimum) Level AA
